documentation/dev_running_from_source.markdown

```diff
@@ -159,6 +159,25 @@ run the following:
 
     $ lein run -c /path/to/puppetserver.conf
 
+Running the Agent
+-----
+
+Use a command like the one below to run an agent against your running puppetserver:
+
+        puppet agent --confdir ~/.puppetlabs/etc/puppet \
```

Comment:
I think if you specify the --confdir setting that you wouldn't need the --server setting since it should read the value for the server setting that was populated into the ~/.puppetlabs/etc/puppet/puppet.conf file, which should be equal to whatever you used for <MASTERHOST>.  Ideally, you wouldn't need to specify the --confdir setting either for a Puppet v4.x agent since ~/.puppetlabs/etc/puppet should be the default.  Doesn't hurt anything either way, though.